zerver/lib/bugdown/__init__.py

```diff
@@ -1200,6 +1201,17 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in realm_slash_commands:
+                escaped = re.escape(("/" + command).lower())
+                match_re = re.compile(u'(?:%s)%s(?:%s)' %
+                                      (allowed_before_punctuation,
+                                       escaped,
+                                       allowed_after_punctuation))
+                if re.search(match_re, content):
+                    current_message.triggered_slash_commands.add(command)
+            if hasattr(current_message, 'triggered_slash_commands') and \
```

Comment:
When would `hasattr(current_message, 'triggered_slash_commands')` return `False`?  If this is the case, would line 1211 (`current_message.triggered_slash_commands.add(command)` produce an error?